TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/modprogression/6moontemple.json

```
@@ -0,0 +1,38 @@
+{
+    "name": "月宫",
+    "icon": "theaurorian:moontemplekey",
+    "category": "2modprogression",
+    "sortnum": "6",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "image",
+            "title": "月宫",
+            "text": "",
+            "images": [
+                "theaurorian:patchouli_books/the_aurorian_guide/en_us/entries/aurorianbasics/moontemple.png"
+            ]
+        },
+        {
+            "type": "text",
+            "text": "$(#008cff)月宫$()是浮空的岛屿，它们只会像$(#008cff)符石地牢$()一样，沿 x 轴和 z 轴生成。有一条小路带你上去。这里是$(#8800ff)皎月女王$()的家！$(br2)月宫附近还有其它的浮空岛屿，重要的合成材料$(#008cff)皎月宝石$()便会在上面生成。"
+        },
+        {
+            "type": "crafting",
+            "title": "月宫钥匙",
+            "recipe": "theaurorian:items/special/moontemplekey"
+        },
+        {
+            "type": "spotlight",
+            "title": "暗黑护符",
+            "item": "theaurorian:darkamulet",
+            "text": "$(#8800ff)蛛母$()会掉落能通往下一座地牢的$(#008cff)暗黑护符$()。$(br2)$(#008cff)月宫$()的内部有两座塔。塔顶会有$(#008cff)月宫内室钥匙碎片$()。"
```

|Feedback Priority|
|---|
|⛰ Mountain / **Blocking and requires immediate action**|

```suggestion
            "text": "$(br2)$(#008cff)月宫$()的内部有两座塔。塔顶会有$(#008cff)月宫内室钥匙碎片$()。"
```
删掉的这一句需要实测。